nextflow.config

```diff
@@ -67,11 +67,12 @@ profiles {
         cleanup                = false
         nextflow.enable.configProcessNamesValidation = true
     }
-    wave {
-        wave.enabled = true
-        wave.strategy = ['conda', 'container', 'dockerfile', 'spack']
-        wave.build.repository = 'quay.io/seqeralabs/nf-aggregate'
-        wave.freeze = true   
+    seqera_containers {
+        apptainer.ociAutoPull   = true
+        singularity.ociAutoPull = true
+        wave.enabled            = true
+        wave.freeze             = true
+        wave.strategy           = 'conda'
```

Comment:
If we are using Wave behind the scenes to get the container URI, do I need to move these out of a profile and into the top-level config so they are always on?

Reply:
It's keep into a separate profile